src/utils.ts

```diff
@@ -64,12 +65,30 @@ const sanitizeMessage = (message: string) => message.trim().replace(/[\n\r]/g, '
 
 const promptTemplate = 'Write an insightful but concise Git commit message in a complete sentence in present tense for the following diff without prefacing it with anything:';
 
+const getTranslatedPrompt = () => {
+	const validLangs = ['en', 'es', 'jp', 'zh', 'de', 'fr', 'it', 'pt'];
+	const program = new Command();
+
+	program.option('-l, --lang <string>', 'ISO code language (2-chars) to use for generating the commit message', 'en');
```

Comment:
Add it here:
https://github.com/Nutlope/aicommits/blob/68fc8ad736e38a9497e94236800d32c9264576d7/src/cli.ts#L23-L30

Reply:
Done!